internal/notify/client.go

```diff
@@ -165,10 +185,33 @@ func (c *Client) SendActorSMS(ctx context.Context, to, lpaUID string, sms SMS) e
 	return nil
 }
 
-func (c *Client) newRequest(ctx context.Context, url string, wrapper any) (*http.Request, error) {
+func (c *Client) recentlySent(ctx context.Context, ref string) (bool, error) {
+	req, err := c.newRequest(ctx, http.MethodGet, "/v2/notifications?reference="+ref, nil)
+	if err != nil {
+		return false, err
+	}
+
+	resp, err := c.do(req)
+	if err != nil {
+		return false, err
+	}
+
+	for _, notification := range resp.Notifications {
+		if (notification.Status == "sending" || notification.Status == "delivered") &&
+			notification.CreatedAt.After(c.now().Add(-allowResendAfter)) {
```

Comment:
I'm kind of scared of setting a blanket policy on preventing resends (which would be 7 days based on what you can query in notify), so I'm choosing a 10 minute window as that will prevent problems with people resubmitting the form

Reply:
Sensible 👍